adminutils/adminutils.py

```diff
@@ -237,6 +269,11 @@ async def emote_steal(self, ctx, name: str, message_id: discord.Message, *roles:
             return
         except discord.HTTPException as e:
             await ctx.send(chat.error(_("An error occurred on adding an emoji: {}").format(e)))
+        else:
+            msg = _("{em} created with the name `{em.name}`".format(em=em))
+            msg += _(" and locked to the roles {}.".format(chat.humanize_list(role.name for role in roles)) if roles else ".")
```

Comment:
```suggestion
            msg += _(" and locked to the roles {}.".format(chat.humanize_list([role.name for role in roles])) if roles else ".")
```

`TypeError: object of type 'generator' has no len()`